graph/src/util/stable_hash_glue.rs

```diff
@@ -0,0 +1,82 @@
+use stable_hash::{StableHash, StableHasher};
+use stable_hash_legacy::prelude::{
+    StableHash as StableHashLegacy, StableHasher as StableHasherLegacy,
+};
+
+/// Implements StableHash and StableHashLegacy. This macro supports two forms:
+/// Struct { field1, field2, ... } and Tuple(transparent). Each field supports
+/// an optional modifier. For example: Tuple(transparent: AsBytes)
+#[macro_export]
+macro_rules! _impl_stable_hash {
```

Comment:
I also think this macro would probably be more suited to be added to the stable-hash crate since it mostly deals with the details from that crate and how it implements this trait for structs. Is there a reason why we shouldn't move it there and add some test coverage for it? Possibly even backport it to the old version of stable-hash is that makes sense? In that case I guess this would just rely those two? Thoughts?

Reply:
We now have macros in both the fast and legacy stable-hash crates, and a graph-node macro that delegates to the other two.